src/kernel.cr

```diff
@@ -470,15 +470,15 @@ class Process
   end
 end
 
-{% unless flag?(:win32) %}
-  # Background loop to cleanup unused fiber stacks.
-  spawn do
-    loop do
-      sleep 5
-      Fiber.stack_pool_collect
-    end
+# Background loop to cleanup unused fiber stacks.
+spawn do
+  loop do
+    sleep 5
```

Comment:
sleep isn't implemented on windows?

Reply:
It is in `windows_stubs.cr`.

Reply:
`sleep` can't be implemented on win32 without an event loop and timers; i.e. chicken-egg.